Source/Details/ASElementMap.h

```diff
@@ -31,6 +31,11 @@ NS_ASSUME_NONNULL_BEGIN
 AS_SUBCLASSING_RESTRICTED
 @interface ASElementMap : NSObject <NSCopying, NSFastEnumeration>
 
+/**
+ * The total number of elements in this map.
+ */
+@property (readonly) NSUInteger count;
```

Comment:
This is required by the new implementation of `ASArrayByFlatMapping`, in addition to NSFastEnumeration conformance.